templates/takeovers/_compliance-webinar.html

```diff
@@ -0,0 +1,26 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-deep js-takeover is-dark p-takeover--compliance {% if not default %}u-hide{% endif %}"
+    {% if default %}data-default="true" {% endif %}>
+    <div class="row u-vertically-center">
+        <div class="col-7 u-fade-left--medium">
+            <h1 style="font-weight: 100">Securing the enterprise</h1>
+            <h4 class="u-no-padding--top  u-sv3">Learn how Ubuntu’s trusted security and compliance solutions are at the forefront of solving regulatory issues across a wide range of industries.</h4>
```

Comment:
Please make this a `p` with `p-heading--four`

Reply:
@anthonydillon the change has been made now